scripts/cleanup.sh

```diff
@@ -203,6 +204,33 @@ function delete_keys() {
   RUN_IBMCLOUD pi key-delete "${INFRA_ID}-key"
 }
 
+function delete_vpcs() {
+  echo "Deleting VPC and VPC Subnet"
+  RUN_IBMCLOUD is vpcs --output JSON
+  vpc_id="$(echo "${CMD_OUT}" | jq -r ".[]|select(.name==\"powervs-ipi\").crn")"
```

Comment:
Why is this `powervs-ipi` only? The new subnets will be part of `vpc-${INFRA_ID}` right?

Reply:
will fix